backend/routes/index.ts

```diff
@@ -1,10 +1,13 @@
 import express from "express";
 import FilesController from "../controllers/FilesController";
 import UserController from "../controllers/UserController";
+import LocationController from "../controllers/LocationsController";
 
 const router = express.Router();
 
 router.get("/users", UserController.getUsers);
+router.get("/locations", LocationController.getLocationId);
```

Comment:
method get does not have body, therefore method getLocationId can not get id prop from it.
solvation: use params like -
"/locations/:id"
and req.params['id'] in getLocationId method [row #7]